app/models/provider_interface/provider_applications_page_state.rb

```diff
@@ -125,22 +125,33 @@ def status_filters
     end
 
     def provider_filters_builder
-      input_config = @application_choices.map do |choice|
-        provider = choice.provider
-
+      input_config = ProviderOptionsService.new(provider_user).providers.map do |provider|
         {
           type: 'checkbox',
           text: provider.name,
           name: provider.id.to_s,
         }
       end
 
-      provider_filters = {
+      {
         heading: 'provider',
-        input_config: input_config.uniq,
+        input_config: input_config,
       }
+    end
+
+    def accrediting_provider_filters_builder
```

Comment:
This should be `accredited provider`: https://ukgovernmentdfe.slack.com/archives/CAHBLU6ET/p1585561134106700

Reply:
just saw that Slack thread will update terminology in code. As a separate exercise it might be worth correcting `providers.accrediting_provider_id` in the DB schema but I'll leave that for now.

Reply:
yes, I agree :+1:

Reply:
https://trello.com/c/3gHEY19s/1828-rename-accrediting-provider-to-accredited-provider-in-db-schema